src/css/styles.css

```diff
@@ -1,71 +1,66 @@
 body {
-  /* background-image: url(https://images.trvl-media.com/hotels/12000000/11620000/11615100/11615006/416e4a41_z.jpg); */
   background-repeat: no-repeat;
   background-size: cover;
-  margin: 0;
+  font-family: 'Comic Sans MS';
   height: 100vh;
+  margin: 0;
   width: 100%;
 }
 
-/* *****Login Page***** */
+/* LOGIN PAGE */
+
 .login-page{
-  /* border: 2px solid green; */
-  height: 100%;
-  width: 100%;
-  margin: 0;
+  align-items: center;
   display: flex;
   flex-direction: column;
+  height: 100%;
   justify-content: center;
-  align-items: center;
+  margin: 0;
+  width: 100%;
 }
 
 .greeting{
-  /* border: 2px solid purple; */
-  height: 40%;
+  align-items: center;
   display: flex;
   flex-direction: column;
+  height: 40%;
   justify-content: center;
-  align-items: center;
 }
 
 .login{
   text-shadow: 2px 2px #4a4848;
 }
 
 .login-title{
-  /* border: 2px solid blue; */
   font-size: 100px;
-  text-shadow: 2px 2px #4a4848;
-  text-align: center;
   size: 5em;
+  text-align: center;
+  text-shadow: 2px 2px #4a4848;
 }
 
 .slogan{
-  /* border: 2px solid red; */
-  margin: 0;
   font-size: 30px;
+  margin: 0;
```

Comment:
You can zero out the margin at the very top of the style sheet, that way it's the default case - might save lines of code!